.github/workflows/check.yaml

```diff
@@ -0,0 +1,32 @@
+name: Check workflow running linter, unit and functional tests
```

Comment:
We also need to make changes in `sonar.yaml` ([L#9](https://github.com/canonical/prometheus-hardware-exporter/blob/main/.github/workflows/sonar.yaml#L9) and[ L#17](https://github.com/canonical/prometheus-hardware-exporter/blob/main/.github/workflows/sonar.yaml#L17)) to track this new workflow name.